src/Sulu/Bundle/MediaBundle/Search/Subscriber/StructureMediaSearchSubscriber.php

```diff
@@ -81,7 +81,9 @@ public function handlePreIndex(PreIndexEvent $e)
         $subject = $e->getSubject();
         $evaluator = $e->getFieldEvaluator();
 
-        if (false === $metadata->getClassMetadata()->reflection->isSubclassOf(StructureBehavior::class)) {
+        if (
+            !$e->getSubject() instanceof StructureBehavior
+            && StructureBehavior::class !== $metadata->getName()) {
```

Comment:
Can you move the closing bracket from the if into a separate line? This way it reads a little bit strange.